services/auths.js

```diff
@@ -23,3 +24,23 @@ exports.verifyEmail = async id => {
     return error.message;
   }
 };
+
+exports.loginUser = async (userData) => {
+  const { email, password, username } = userData;
+
```

Comment:
I think it might be best and also makes for a great user experience to have someone either log in with their email or username. Very few apps these days implement a sole login feature.

Reply:
thought we had this changed, can we remove the username @Uzoamaka126